dracut.sh

```diff
@@ -1282,7 +1282,7 @@ if [[ -f $dracutbasedir/dracut-version.sh ]]; then
 fi
 
 if systemd-detect-virt -c &> /dev/null; then
-    export DRACUT_NO_MKNOD=1 DRACUT_NO_XATTR=1
+    export DRACUT_NO_MKNOD=1 DRACUT_NO_XATTR=${DRACUT_NO_XATTR:-1}
```

Comment:
Why are we defaulting to NO_XATTR here?

Reply:
Here is the PR where that change landed https://github.com/dracutdevs/dracut/pull/1499 and referenced by https://github.com/dracutdevs/dracut/issues/890 .

For now I will just take this change out of this PR as it seems this needs more discussion.